js/options.js

```diff
@@ -9,20 +9,21 @@ $(function() {
         }
     });
     $('#save').on('click', function() {
-        getTenants(function(tenants) {
-            for (var tenantCode in tenants) {
-                if (tenantCode === $('#tenantCode').val()) {
-                  saveActiveTenant(tenants[tenantCode], function() {
-                    // Update status to let user know options were saved.
-                    $('#status').html('<div class="alert alert-success">' + chromeOrBrowser().i18n.getMessage('optionsSettingsSaved') + '</div>');
-                    $('#optionsHelp').addClass('hidden');
-                    setTimeout(function() {
-                      $('#status').textContent = '';
-                    }, 750);
-                  });
+        if ($('#specifyTenant:selected').length > 0) {
```

Comment:
If we're already in league with jQuery, would `is(':selected')` be a shortcut for this?

Reply:
possibly, this check was already used within the code so I preferred that (knowing it had already been tested in all the browsers).